mmda/types/document_elements.py

```diff
@@ -1,11 +1,16 @@
+"""
+
+
+"""
+
+# TODO[kylel] not sure this class needs to exist; seems extra boilerplate for no benefit
```

Comment:
I think it would help:
1. for extensibility for future updates that stores token-level bbox
2. for indexing tokens across pages: now we support something like `tokens[page_id, start:end]`

But yeah.. could be removed if you think that's more straightforward